main.go

```diff
@@ -420,13 +444,41 @@ func main() {
 	}
 	svc := s3.NewFromConfig(cfg)
 
+	promRegistry := newStatsRegistry(*metricsAddress)
+
+	requestsMetric := prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Name: "ctile_requests",
+			Help: "total number of requests, by result and source",
+		},
+		[]string{"result", "source"},
```

Comment:
Add the log/shard to the labels for each of these metrics so support the use-case of a single ctile for multiple shards.

Reply:
Currently we're planning on a "one ctile, one CT log (aka one CT log shard)" deployment model.